Mathlib/Order/Directed.lean

```diff
@@ -219,6 +224,16 @@ protected theorem IsMax.isTop [IsDirected α (· ≤ ·)] (h : IsMax a) : IsTop
   h.toDual.isBot
 #align is_max.is_top IsMax.isTop
 
+lemma DirectedOn.isBot_of_isMin {s : Set α} (hd : DirectedOn (· ≥ ·) s)
```

Comment:
I hope the capitalization is correct ... Note `IsBot` and `IsMin` aren't actually involved; the lemma is so named because it's analogous to `IsMin.isBot`.

Reply:
I would say that if they're not about the actual definitions themselves it should be lower case. Like "conservative" and "Conservative"

Reply:
Okay!